brave/brave-core
# Implement Brave Ads field trials

Resolves https://github.com/brave/brave-browser/issues/16213

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm ads are not served if an ad previously converted for the same creative set
- Confirm ads are not served if an ad was dismissed within the last 48 hours for the same campaign
- Confirm ads are not served if an ad was transferred (landed) within the last 48 hours for the same campaign
- Confirm ads are not served if the browser is in fullscreen mode
- Confirm ads are not served if media is playing within the active tab
- Confirm ads are not served if the browser is inactive in the background, with the exception of Android if background notifications are enabled within the UI
- Confirm ads are not served if we do not have an active network connection
